app/controllers/application_controller.rb

```diff
@@ -291,10 +291,6 @@ def user_fully_authenticated?
       two_factor_enabled?
   end
 
-  def confirm_user_is_not_suspended
```

Comment:
is this method still used?

Reply:
oh whoops it still is as a before filter in a few spots 😬

Reply:
removed in 6bebfb514efac1481b666e594cea29187c689cef

Reply:
ok so after chatting with @aduth we think it's good to have this filter too, I brought it back in dfe253b29c023b996f4b9dabd7e297a7aba34098

Reply:
It could probably live in AccountsController instead of ApplicationController? Since it's only used once in that controller.

Reply:
yes good call